src/world/Explosion.php

```diff
@@ -150,13 +163,35 @@ public function explodeB() : bool{
 		$yield = min(100, (1 / $this->radius) * 100);
 
 		if($this->what instanceof Entity){
-			$ev = new EntityExplodeEvent($this->what, $this->source, $this->affectedBlocks, $yield);
+			$ev = new EntityExplodeEvent($this->what, $this->source, $this->affectedBlocks, $yield, $this->fireIgnitions);
+
+			$ev->setIgnitions($this->fireIgnitions);
```

Comment:
There is no point to call `$ev->setIgnitions()` because you already passed them in the event constructor